.travis.yml

```diff
@@ -1,71 +1,71 @@
 services:
-- docker
+  - docker
```

Comment:
don't try to reformat to minimize the unnecessary changes for easier, faster reviewing.